src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -649,8 +649,7 @@ TransactionManagerRef make_transaction_manager(
     roll_start = 0;
   } else {
     // FIXME: get from runtime configration instead of static defaults
-    roll_size = journal::CircularBoundedJournal::mkfs_config_t
-                       ::get_default().total_size;
+    roll_size = journal::DEFAULT_TEST_CBJOURNAL_SIZE;
```

Comment:
Now we can load roll_start and roll_size from `RBMDevice` instead, which implements the FIXME.

Reply:
Done.